packages/table/src/interactions/reorderable.tsx

```diff
@@ -130,6 +137,13 @@ export class DragReorderable extends React.Component<IDragReorderable, {}> {
         } else {
             // select the new region to avoid complex and unintuitive UX w/r/t the existing selection
             this.props.onSelection([region]);
+            const focusCellCoordinates = Regions.getFocusCellCoordinatesFromRegion(region);
```

Comment:
We're inconsistent about `*focusedCell*` vs `*focusCell*`. I've opted to move to the former in my work; you okay with that? If so, let's change this to `focusedCellCoordinates`.

Reply:
Done